src/AutoMerge/new-package.jl

```diff
@@ -163,7 +179,9 @@ function pull_request_build(api::GitHub.GitHubAPI,
                               g5,
                               g6,
                               g7,
-                              g8]
+                              g8,
+                              g_distance,
```

Comment:
Following the convention in the rest of the package, can you number the checks in an increasing order.

The number of each check should correspond to the numbered list in the comment at the top of the function (I.e. the comment starting at line 15 of this file).

Reply:
Ok, fixed! I did that to avoid having to renumber the later checks, but I see I missed putting the new g's and m's into the right lists, which might be harder to miss with renumbering.